client_server_info_impl.go

```diff
@@ -140,6 +140,13 @@ func (c *client) echo(ctx context.Context) error {
 	if err != nil {
 		return WithStack(err)
 	}
+
+	// Velocypack requires non-empty body for versions < 3.11.
+	req, err = req.SetBody("echo")
```

Comment:
Is this "echo" value somehow a specific keyword? Maybe it worth moving into constant and re-use that?

Reply:
no, `echo` is just an any body. It can be any non empty string

Reply:
@nikita-vanyasin This `echo` method is like `echo` in bash - it returns the same request as you send